validator/keymanager/wallet_test.go

```diff
@@ -95,3 +95,61 @@ func TestMultiplePassphrases(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvPassphrases(t *testing.T) {
+	path := SetupWallet(t)
+	defer func() {
+		if err := os.RemoveAll(path); err != nil {
+			t.Log(err)
+		}
+	}()
+
+	if err := os.Setenv("NEITHER", "neither"); err != nil {
```

Comment:
Could you unset the environment variables after the test is done ?

Reply:
Thanks, added.  Also altered the names to use a test-specific prefix to avoid any chance of clash.